README.md

```diff
@@ -1,2 +1,33 @@
-# My-first-microverse-project
-This is my first project in microverse.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+### Hello Microverse Folks
+
+> This project is part of the first module of Microverse, It only contains an html file with a css file.
+
+
```

Comment:
- [ ] Kindly make sure your README has a detailed description telling what the project is about  :smile: 
Hint: You can refer to the project description [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Estimated%20time%3A%200.5h-,Description,-In%20this%20project)